xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -715,6 +715,22 @@ JSONRPC_STATUS CPlayerOperations::Open(const std::string &method, ITransportLaye
     CApplicationMessenger::GetInstance().PostMsg(TMSG_EXECUTE_BUILT_IN, -1, -1, nullptr, "playercontrol(partymode(" + parameterObject["item"]["partymode"].asString() + "))");
     return ACK;
   }
+  else if (parameterObject["item"].isMember("broadcastid"))
+  {
+    const std::shared_ptr<CPVREpgInfoTag> epgTag =
+        CServiceBroker::GetPVRManager().EpgContainer().GetTagById(
+            nullptr,
+            static_cast<unsigned int>(parameterObject["item"]["broadcastid"].asUnsignedInteger()));
+
+    if (!epgTag || !epgTag->IsPlayable())
+      return InvalidParams;
+
+    if (!CServiceBroker::GetPVRManager().GUIActions()->PlayEpgTag(
+            std::make_shared<CFileItem>(epgTag)))
```

Comment:
Use 'CServiceBroker::GetPVRManager().GUIActions()->PlayMedia()', passing the epgtag boxed in an item, like we do for recordings and channels?

Reply:
Do we get resume for free then? I followed the path we use for playing an epg tag currently. Should this be different to that?

Reply:
Could you try whether PlayMedia works? It long time ago I implemented this, so I'm not sure.

Reply:
From the code, PlayMedia seems to support recordings and channels only.

Reply:
Play media works if I add the handling for an epg tag.

But as I already pass an epg tag file item and call PlayEpgTag anyway it just seems redundant.